src/Main.java

```diff
@@ -1,278 +1 @@
-package com.company;
 
-import javax.swing.text.StyledEditorKit;
-import java.util.Scanner;
-import java.io.File;
-import java.io.*;
-import java.util.Queue;
-import java.util.*;
-
-class Cell{
-    int x;
-    int y;
-    int gCost;
-    int hCost;
-    Cell parent;
-
-    public Cell(int x, int y, Cell parent){
-        this.x = x;
-        this.y = y;
-        this.parent = parent;
-    }
-
-    public Cell(int x, int y, Cell parent, int gCost, int hCost){
-        this.x = x;
-        this.y = y;
-        this.parent = parent;
-        this.gCost = gCost;
-        this.hCost = hCost;
-    }
-
-    public Cell getParent(){
-        return this.parent;
-    }
-}
-
-class MazeSolver {
-
-    //4 possible moves
-    static int[] move_x = {1, -1, 0, 0};
-    static int[] move_y = {0, 0, 1, -1};
-
-    // Using DFS algorithm, does not guarantee shortest path
-    static Boolean MazeBackTracker(char[][] maze, Cell cell, int m, int n){
```

Comment:
You should consider your naming conventions for classes and functions. The normally used Java one is:
CapitalCase - classes
camelCase - functions

So, in this case, it should be 'mazeBackTracker' (which is also a bad name - functions should be actions or verbs)